File: 2-ad-hoc/3048/3048.cs
using System;

class URI {
    static void Main(string[] args) {
        int size = Int32.Parse(Console.ReadLine());
        int[] arr = new int[size];
        int last = 0;
        int consec = 0;
        for (int i = 0; i < size; i++) {
            arr[i] = Int32.Parse(Console.ReadLine());
            if (arr[i] != last) consec++;
            last = arr[i];
        }
        Console.WriteLine(consec);
    }
}

File: 2-ad-hoc/3048/3048.php
<?php
$size = (int)trim(fgets(STDIN));
$arr = array();
$last = 0;
$consec = 0;
for ($i = 0; $i < $size; $i++) { 
    $arr[$i] = (int)trim(fgets(STDIN));
    if ($arr[$i] != $last) $consec++;
    $last = $arr[$i];
}
echo $consec.PHP_EOL;
?>